packages/presentation/package.json

```diff
@@ -42,6 +42,7 @@
     "@hcengineering/login": "^0.6.1",
     "@hcengineering/image-cropper": "^0.6.0",
     "@hcengineering/client": "^0.6.6",
+    "@hcengineering/tracker": "^0.6.1",
```

Comment:
This is not allowed, presentation is generic package, it should not know about tracker.

Reply:
Fixed. Moved to tracker-resource.